sanity/services/csr-query.ts

```diff
@@ -0,0 +1,9 @@
+export const csrQuery = `*[_type == "csr"]{
+    _id,
+    title,
+    titleColor,
+    subtitle,
+    description,
+    images,
```

Comment:
which we use for generate the url image ?
using groq or using helper in code ?

please standarize for that

Reply:
Sure